src/dotnet/README.md

```diff
@@ -31,27 +31,25 @@ This Feature installs the latest .NET SDK, which includes the .NET CLI and the s
 Installing only the latest .NET SDK version (the default).
 
 ``` json
-{
 "features": {
-    "ghcr.io/devcontainers/features/dotnet:2": "latest" // or "" or {}
+    "ghcr.io/devcontainers/features/dotnet:2": "latest"
```

Comment:
Can we keep other changes to README file as is? I think the current changes makes sense.

Reply:
Sure, I mainly wanted to fix this code highlighting issue

![image](https://github.com/devcontainers/features/assets/1583241/bf259ed8-9a97-41ca-8eff-5edd0067e2a9)

Fixed it by changing the language to `jsonc` and also removed the unnecessary brace at the top.